triton/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=triton
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MPL-2.0')
+pkg_deps=(core/node)
+pkg_build_deps=(core/jq-static core/sed)
+pkg_bin_dirs=(bin)
+pkg_description="Joyent Triton CLI and client"
+pkg_upstream_url="https://www.joyent.com/triton"
+
+pkg_version () {
+  npm view triton --json | jq --raw-output .version
```

Comment:
`o_o` well kind of need like some kind of "sha-ca".  Can't really get your sha's from the same source as your package because what we have to assume is that website could be compromised and I'll need to get a copy of the sha's validate against from somewhere I trust.

 I'm all for automation - just need to balance whether it makes sense to automate third-party packages or if the packages not so frequently that this type of automation just introduces security risk to habitat's users.

Again super paranoid.